enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -9,7 +9,7 @@
   "main": "./dist/main.bundle.js",
   "types": "./dist/enterprise/frontend/src/embedding-sdk/index.d.ts",
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": ">17.0.2 <=18",
```

Comment:
<=18 works with other minor versions too. You can test this syntax with https://semver.npmjs.com/

Reply:
@WiNloSt I wonder if you need `>=` instead

with >17.0.2 <=18

![CleanShot 2567-05-09 at 21 16 33@2x](https://github.com/metabase/metabase/assets/4714175/e939a4bc-b5fb-4186-b53a-d7eba6bc531f)

with >=17.0.2 <=18

![CleanShot 2567-05-09 at 21 17 05@2x](https://github.com/metabase/metabase/assets/4714175/9a55622d-5bd2-4bce-b150-1a7648368d5c)

Reply:
Thanks! I failed miserably. I only tested React 18 locally, but didn't see that this site doesn't even include React 17 versions. I changed it to `>=17 <=18` for simplicity. Thanks for being vigilant!